string id
float32 sec_mark
int16 message_count
float32 latitude
float32 longtitude
float32 elevation
float32 pos_accuracy_semi_major
float32 pos_accuracy_semi_minor
float32 pos_accuracy_orientation
float32 lateral_acceleration
float32 longitudinal_acceleration
float32 vert_acceleration
float32 yaw_acceleration
uint8 transmission_state
uint8 response_type
uint8 light_use
uint8 siren_use
uint8 events
uint8 lights
uint8 confidence_position
uint8 confidence_elevation
uint8 vehicle_class
uint8 vehicle_fuel_type
float32 heading
float32 speed
int16 angle
float32 vehicle_width
float32 vehicle_lenth
float32 vehicle_height
uint8 brake_padel
uint8 wheel_brakes
uint8 traction
uint8 abs
uint8 scs
uint8 brake_boost
uint8 aux_brakes


























